app-android-xamarin/Moodis/Moodis/Resources/layout/holder_members_entry.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout 
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    xmlns:card_view="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <android.support.v7.widget.CardView
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        card_view:cardElevation="@dimen/cardElevation"
+        card_view:cardUseCompatPadding="true"
+        card_view:cardCornerRadius="@dimen/cardCornerRadius">
+
+        <android.support.constraint.ConstraintLayout 
```

Comment:
Idk, but possibly this layout is not really needed, as it contains only one element

Reply:
do you want to just use one layout for three activities?

Reply:
This layout contains only textview. constraint layout != activity